src/router/projects.js

```diff
@@ -587,4 +587,55 @@ export const MY_PROJECT_VIEWS = [
       },
     ],
   },
+  {
+    path: ':id/institutions/',
+    name: 'MyProjectInstitutionsView',
+    meta: {
+      itemName: 'institutions',
+      breadcrumbs: [
+        {
+          label: 'institutions',
+          to: 'MyProjectInstitutionsView',
+        },
+      ],
+    },
+    component: () =>
+      import(
+        /* webpackChunkName: "unpublished" */ '@/views/project/institutions/ListView.vue'
+      ),
+  },
+  {
+    path: ':id/institutions/assign',
+    name: 'MyProjectInstitutionsAssignView',
+    meta: {
+      itemName: 'institutions',
+      breadcrumbs: [
+        {
+          label:'institutions',
+          to: 'MyProjectInstitutionsView',
+        },
+      ],
+    },
+    component: () =>
+      import(
+          /* webpackChunkName: "unpublished" */ '@/views/project/institutions/AssignView.vue'
+      ),
+  },
+  {
```

Comment:
Don't create a View for the deleting views. Instead, let's do a dialog instead which is similar to the "Views for Media" page.